### PR TITLE
pkcsconf: Check for unrecognized or left-over options

### PR DESCRIPTION
**Pull request by ifranzki**
Report left-over options that were not processed by getopt() as error, and do not silently ignore them.

### CODE REVIEW

**Review by vkanjur — APPROVED** [review]
Approved

**Review by jschmidb — APPROVED** [review]
lgtm

File: usr/sbin/pkcsconf/pkcsconf.c [hunk 1]
```diff
@@ -991,14 +979,11 @@ CK_RV init_token(int slot_id, CK_CHAR_PTR pin)
...
-            printf("Incorrect PIN Entered.\n");
-            fflush(stdout);
-        } else {
-            printf("Error initializing token: 0x%lX (%s)\n", rc,
-                   p11_get_ckr(rc));
-            fflush(stdout);
-        }
+        if (rc == CKR_PIN_INCORRECT)
+            warnx("Incorrect PIN Entered.");
+        else
```
Comment (jschmidb) [inline usr/sbin/pkcsconf/pkcsconf.c:984]: ... entered (?)

**Review by jschmidb — APPROVED** [review]

File: usr/sbin/pkcsconf/pkcsconf.c (hunk 1 above)
Comment (ifranzki) [reply usr/sbin/pkcsconf/pkcsconf.c]: yes, fixed those, too.
No need to re-approve :-)

**Pull request merged**